src/lib/skilling/skills/herblore/mixables/crush.ts

```diff
@@ -90,6 +90,16 @@ const Crush: Mixable[] = [
 		bankTimePerPotion: 0,
 		wesley: true
 	},
+	{
+		item: getOSItem('Lava scale shard'),
+		aliases: ['lava dragon scale', 'lava dragon dust', 'lava dragon', 'lava shard'],
+		level: 1,
+		xp: 0,
+		inputItems: new Bank({ 11_992: 1 }),
```

Comment:
Whys this using the item id?

Reply:
No real reason. I replaced the ID with the item name, I also changed the code in herbloreAcitivity to also use the item name